components/proxy/src/main/java/com/hotels/styx/Version.java

```diff
@@ -74,7 +80,28 @@ public String releaseVersion() {
         return releaseTag.substring(firstDot + 1);
     }
 
+    public Optional<Integer> buildNumber() {
```

Comment:
Is this used anywhere beyond tests?

Reply:
umm the changes to CoreMetrics were uncommited... it should be fixed now.